tox.ini

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Canonical Limited
+# See LICENSE file for licensing details.
+
+[tox]
+no_package = True
+skip_missing_interpreters = True
+env_list = lint, unit
+
+[vars]
+application = data_platform_helpers
+src_path = {tox_root}/data_platform_helpers
+tests_path = {tox_root}/tests
+all_path = {[vars]src_path} {[vars]tests_path}
+
+[testenv]
+allowlist_externals =
+    /bin/bash
+    poetry
+
+set_env =
+    PYTHONPATH = {tox_root}/lib:{[vars]src_path}
+    PYTHONBREAKPOINT=ipdb.set_trace
+    PY_COLORS=1
+pass_env =
+    PYTHONPATH
+    CHARM_BUILD_DIR
+    MODEL_SETTINGS
+
+[testenv:format]
+description = Apply coding style standards to code
+commands =
+    poetry install
+    poetry export -f requirements.txt -o requirements.txt --without-hashes
+
+    poetry install --only format
+    poetry run ruff check --fix {[vars]all_path}
+    poetry run black {[vars]all_path}
```

Comment:
Do we want to use ruff for formatting?

Reply:
tbh I am not sure, I would be curious what others might think

Reply:
Since it's already there for linting, I am +1

Reply:
+1 to that. It's the trend everywhere on charming world

Reply:
Added ruff formatting